pkg/controllers/custompackage/controller.go

```diff
@@ -206,9 +206,13 @@ func (r *Reconciler) reconcileGitRepo(ctx context.Context, resource *v1alpha1.Cu
 				Type: "local",
 				Path: absPath,
 			},
-			GitURL:         resource.Spec.GitServerURL,
-			InternalGitURL: resource.Spec.InternalGitServeURL,
-			SecretRef:      resource.Spec.GitServerAuthSecretRef,
+			Provider: v1alpha1.Provider{
+				Name:             v1alpha1.GitProviderGitea,
+				GitURL:           resource.Spec.GitServerURL,
+				InternalGitURL:   resource.Spec.InternalGitServeURL,
+				OrganizationName: v1alpha1.GiteaAdminUserName,
```

Comment:
this is a bit confusing here that we have an variable `GiteaAdminUserName` as the `OrganizationName` we probably can have a better variable name or make some clarification here

Reply:
This is how it's set up right now and this will be changed in the future. The primary purpose of this PR is to keep the current default behaviour but allow for modifications in the future.